content/topics/clean-code/python/_index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Clean Code for Python
+ready: True
+---
+
+# Clean Code for Python
+
+## Tabs or Spaces
+
+`4 spaces == 1 tab`
+
+Spaces are the preferred indentation method.
+Tabs should be used solely to remain consistent with code that is already indented with tabs.
+Python 3 **DISALLOWS** mixing the use of tabs and spaces for indentation.
+Python 2 code indented with a mixture of tabs and spaces should be converted to using spaces exclusively.
+
+Yes:
+
+    # Aligned with opening delimiter.
+    foo = long_function_name(var_one, var_two,
+                            var_three, var_four)
+
```

Comment:
Just install the `black` autoformatter and let it manage your indentation for you. It is really the best thing. You can set p vscode and other editers to automatically make your code complient and beautiful whenever you hit save. 

Let black focus on nitty gritty details, let coders focus on code